src/plugins/discourse/references.test.js

```diff
@@ -106,42 +112,79 @@ describe("plugins/discourse/references", () => {
         hyperlinkToReference[hyperlink] = linksToReferences([hyperlink])[0];
       }
       expect(hyperlinkToReference).toMatchInlineSnapshot(`
-        Object {
-          "https://discourse.sourcecred.io/t/experiment-sourcecred-stack-lookup/287/4": Object {
-            "postIndex": 4,
-            "serverUrl": "https://discourse.sourcecred.io",
-            "topicId": 287,
-            "type": "POST",
-          },
-          "https://discourse.sourcecred.io/t/experiment-sourcecred-stack-lookup/287/4?u=decentralion": Object {
-            "postIndex": 4,
-            "serverUrl": "https://discourse.sourcecred.io",
-            "topicId": 287,
-            "type": "POST",
-          },
-          "https://discourse.sourcecred.io/u/decentralion": Object {
-            "serverUrl": "https://discourse.sourcecred.io",
+                Object {
+                  "https://discourse.sourcecred.io/t/experiment-sourcecred-stack-lookup/287/4": Object {
+                    "postIndex": 4,
+                    "serverUrl": "https://discourse.sourcecred.io",
+                    "topicId": 287,
+                    "type": "POST",
+                  },
+                  "https://discourse.sourcecred.io/t/experiment-sourcecred-stack-lookup/287/4?u=decentralion": Object {
+                    "postIndex": 4,
+                    "serverUrl": "https://discourse.sourcecred.io",
+                    "topicId": 287,
+                    "type": "POST",
+                  },
+                  "https://discourse.sourcecred.io/u/decentralion": undefined,
+                  "https://forums.eveonline.com/t/topic/195153": Object {
+                    "serverUrl": "https://forums.eveonline.com",
+                    "topicId": 195153,
+                    "type": "TOPIC",
+                  },
+                  "https://forums.eveonline.com/u/dorian_neil/summary": undefined,
```

Comment:
I feel like absolute URLs should still be supported.

Reply:
Fixed.